src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -366,11 +366,13 @@ private CompilationResult GetCompilation(Func<Compilation, Compilation> compilat
             return result;
         }
 
-        private string GetPathForNewContext(string contextShortTypeName)
+        private string GetPathForNewContext(string contextShortTypeName, string areaName)
         {
+            var areaPath = string.IsNullOrEmpty(areaName) ? string.Empty : Path.Combine("Areas", areaName);
```

Comment:
>areaName [](start = 48, length = 8)

maybe add an extension method ToAreaName() - you repeat this statement in other place too